source/zh-cn/api/index.md

```diff
@@ -24,6 +24,7 @@ hexo.init().then(function(){
 `safe` | 开启安全模式。不加载任何插件。| `false`
 `silent` | 开启安静模式。不在终端中显示任何信息。| `false`
 `config` | 指定配置文件的路径。| `_config.yml`
+`draft` / `drafts`| 是否将草稿加入到文章列表中。<br>例如在 `hexo.locals.get('posts')` 中获取草稿内容 | _config.tml 中 `render_drafts` 的值
```

Comment:
prefer to use _config.yml

Reply:
Fixed in commit https://github.com/hexojs/site/pull/1030/commits/04ff7bb7c12f48eb3b6e4f2a9739d015002328fe